src/deps.deno.ts

```diff
@@ -8,12 +8,24 @@ export type { File } from "https://lib.deno.dev/x/grammy@v1/types.ts";
 
 // Determine whether a file path is absolute
 export { isAbsolute as isAbsolutePath } from "https://deno.land/std@0.135.0/path/mod.ts";
+// Perform a request and returns the raw body
+async function fetchResponseBody(url: string) {
+    const { body } = await fetch(url);
+    if (body === null) throw new Error("Download failed, no response body!");
+    return body;
+}
 // Create a temporary file, and copy a local file to a file path
 export const { makeTempFile: createTempFile, copyFile } = Deno;
+// Streams the repsonse body of a URL
+export async function* streamFile(url: string) {
+    const body = await fetchResponseBody(url);
+    for await (const chunk of body) {
+        yield chunk;
+    }
```

Comment:
```suggestion
    yield* body;
```

Reply:
The whole function is unnecessary. `ReadableStream` is already `AsyncIterable`, the new public method just needs to `yield* await`.